docs/sphinx/user_guide/feature/loop_basic.rst

```diff
@@ -344,22 +344,25 @@ the same team. The ``RAJA::launch`` interface has three main concepts:
     device execution environment, which enables run time selection of
     kernel execution.
 
-  * ``RAJA::LaunchParams`` type. This type takes a number of teams and and a
-    number of threads as arguments.
+  * ``RAJA::LaunchParams`` type. This type takes a number of teams, threads
+    per team, and optionally size of dynamic shared memory in bytes.
 
   * ``RAJA::loop`` template. These are used to define hierarchical
     parallel execution of a kernel. Operations within a loop are mapped to
     either teams or threads based on the execution policy template parameter
     provided.
 
-Team shared memory is available by using the ``RAJA_TEAM_SHARED`` macro. Team
-shared memory enables threads in a given team to share data. In practice,
-team policies are typically aliases for RAJA GPU block policies in the
-x,y,z dimensions, while thread policies are aliases for RAJA GPU thread
-policies in the x,y,z dimensions. In a host execution environment, teams and
-threads may be mapped to sequential loop execution or OpenMP threaded regions.
-Often, the ``RAJA::LaunchParams`` method can take an empty argument list for
-host execution.
+Team shared memory can be allocated by using the ``RAJA_TEAM_SHARED`` macro on
```

Comment:
Probably too late, but were we supposed to rename this to `RAJA_LAUNCH_SHARED`?

Reply:
Good catch!  The macro names should be consistent with the API source code constructs since they are part of the API as well.

Reply:
I was thinking `_TEAM_SHARED` as in memory that is shared between threads in a thread-team which is consistent with the nomenclature used within launch to setup the compute grid. But either way would be okay with me.

Reply:
That's a good point. I forgot about that. Thus, I think it makes more sense the way it is currently named.